doc/radosgw/account.rst

```diff
@@ -0,0 +1,243 @@
+User IDs or tenant names, so Account IDs use a special format to avoid
+ambiguity: the string ``RGW`` followed by 17 numeric digits like
+``RGW33567154695143645``. An Account ID in that format is randomly generated
+upon account creation if one is not specified.
+
+Account IDs are commonly found in the `Amazon Resource Names`_ (ARNs) of IAM
+policy documents. For example, ``arn:aws:iam::RGW33567154695143645:user/A``
+refers to an IAM user named A in that account. The Ceph Object Gateway also
+supports tenant names in that position.
+
+Accounts IDs can also be used in ACLs for a ``Grantee`` of type ``CanonicalUser``.
+User IDs are also supported here.
+
+IAM Policy
+==========
+
+While non-account users are allowed to create buckets and upload objects by
+default, account users start with no permissions at all.
+
+Before an IAM user can perform API operations, some policy must be added to
+allow it. The account root user can add identity policies to its users in
+several ways.
+
+* Add policy directly to the user with the ``iam:PutUserPolicy`` and
+  ``iam:AttachUserPoliicy`` actions.
+
+* Create an IAM group and add group policy with the ``iam:PutGroupPolicy`` and
+  ``iam:AttachGroupPoliicy`` actions. Users added to that group with the
+  ``iam:AddUserToGroup`` action will inherit all of the group's policy.
+
+* Create an IAM role and add role policy with the ``iam:PutRolePolicy`` and
+  ``iam:AttachRolePoliicy`` actions. Users that assume this role with the
+  ``sts:AssumeRole`` and ``sts:AssumeRoleWithWebIdentity`` actions will inherit
+  all of the role's policy.
+
+These identity policies are evaluated according to the rules in
+`Evaluating policies within a single account`_ and
+`Cross-account policy evaluation logic`_.
+
+Principals
+----------
+
+The "Principal" ARNs in policy documents refer to users differently when they
+belong to an account.
+
+Outside of an account, user principals are named by user id such as
+``arn:aws:iam:::user/uid`` or ``arn:aws:iam::tenantname:user/uid``, where
+``uid`` corresponds to the ``--uid`` argument from ``radosgw-admin``.
+
+Within an account, user principals instead use the user name, such as
+``arn:aws:iam::RGW33567154695143645:user/name`` where ``name`` corresponds
+to the ``--display-name`` argument from ``radosgw-admin``. Account users
+continue to match the tenant form so that existing policy continues to work
+when users are migrated into accounts.
+
+Tenant Isolation
+================
+
+Like users, accounts can optionally belong to a tenant for namespace isolation
+of buckets. For example, one account named "acct" can exist under a tenant "a",
+and a different account named "acct" can exist under tenant "b". Refer to
+:ref:`Multitenancy <rgw-multitenancy>` for details.
+
+A tenanted account can only contain users with the same tenant name.
+
+Regardless of tenant, account IDs and email addresses must be globally unique.
+
+Account Management
```

Comment:
@zdover23 of the future: Change all infinitive phrases in this section to imperative sentences. For example: s/To create an account:/To create an account, run a command of the following form:/